resource/js/vocab-search.js

```diff
@@ -225,12 +225,33 @@ const vocabSearch = Vue.createApp({
   template: `
     <div class="d-flex my-auto ms-auto">
       <div class="d-flex justify-content-end input-group ms-auto" id="search-wrapper">
-        <select class="btn btn-outline-secondary dropdown-toggle" data-bs-toggle="dropdown-item"
-          v-model="selectedLanguage"
-          @change="changeContentLangAndReload($event.target.value)"
-          aria-label="Select search language">
-          <option class="dropdown-item" v-for="(value, key) in languageStrings" :value="key">{{ value }}</option>
-        </select>
+
+      <div class="dropdown" id="language-selector">
+        <button class="btn btn-outline-secondary dropdown-toggle"
+           href="#"
```

Comment:
I don't think the `href` attribute is useful for a `button` element?

Reply:
Missed that from the a-element style implementation. I'll fix it and merge.